.github/workflows/issue_bot.yml

```diff
@@ -52,10 +56,24 @@ jobs:
               label => !label.startsWith("Component: ")
             );
 
-            let component_labels = split_body[1]
+            let components = split_body[1]
               .split(',')
-              .map(component => component.trim())
-              .map(component => "Component: " + component);
+              .map(component => component.trim());
+            let moved_components = components.filter(comp => ["Go"].includes(comp));
```

Comment:
How about using `find` not `filter` because this only uses the first found component?